### PR TITLE
Rdf neo4j data access

### PR DESCRIPTION
**Pull request by joluj**

### CODE REVIEW

File: backend/docker/mount/load-dump.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  echo "Found a path to a dump."
+  if [[ ! -f "${DB_PATH}" ]]; then
+    echo "${DB_PATH} does not point to a file. Stopping..."
+    exit 1
+  fi
+  dump_path="${DB_PATH}"
+
+# If DB_URL given, use it
+elif [[ -n "${DB_URL}" ]]; then
+  echo "Found a URL to an dump. Downloading ${DB_URL}."
```
Comment (PianoRollRepresentation) [inline backend/docker/mount/load-dump.sh:17]: change "an" to "a"

**Comment by PianoRollRepresentation** [comment]
The docker/mount/dumps folder (as stated in the README) is missing. The rest looks good

**Comment by joluj** [comment]
> The docker/mount/dumps folder (as stated in the README) is missing. The rest looks good

You cannot commit empty folders

**Comment by CatoLeanTruetschel** [comment]
You can
https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository#answer-932982

**Comment by joluj** [comment]
The folder is optional. I would just leave it like it is.

**Review by PianoRollRepresentation — APPROVED** [review]
Looks fine

**Pull request merged**